### PR TITLE
Assorted docs patches

### PR DESCRIPTION
**Pull request by XhmikosR**
Please check the individual patches, I'm too lazy to split each one to a separate branch. :)

/CC @bardiharborow @Johann-S @mdo

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -54,4 +54,4 @@ cdn:
   jquery:         https://code.jquery.com/jquery-3.2.1.slim.min.js
   jquery_hash:    "sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
   popper:         https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.10.8/umd/popper.min.js
-  popper_hash:    "sha256-n96swYoYKdVyLr5XatK9CzcdyrpAI0xcSlIeWOZeTAI="
+  popper_hash:    "sha384-DnkRdQ7qQnIyQu3l2I77BVrTus+ZPSPbzkftVyG/w9e7UxGp0S0a72dG5FfBY/G3"
```
Comment (Johann-S) [inline _config.yml]: How do you convert `sha256` to `sha384` ? 
If I have to update Popper.js it would be helpful thanks 👍

Comment (XhmikosR) [inline _config.yml]: Just generate the hash for the file with https://srihash.org; it defaults to sha384.

File: _includes/social.html [hunk 2]
```diff
@@ -19,12 +19,17 @@
...
   <meta property="og:url" content="{{ site.url }}">
   <meta property="og:title" content="{{ site.title }}">
   <meta property="og:description" content="{{ site.description }}">
 {% endif %}
-<meta property="og:image" content="{{ site.url }}{{ site.social_image_path }}">
+<meta property="og:image" content="{{ site.url | replace: 'https://', 'http://' }}{{ site.social_image_path }}">
+<meta property="og:image:secure_url" content="{{ site.url }}{{ site.social_image_path }}">
+<meta property="og:image:type" content="image/png">
+<meta property="og:image:width" content="600">
+<meta property="og:image:height" content="600">
```
Comment (mdo) [inline _includes/social.html]: These values need updating—the `bootstrap-social.png` isn't a 600x600 square.

File: _includes/social.html [hunk 3]
```diff
@@ -19,12 +19,17 @@
...
   <meta property="og:title" content="{{ page.title }}">
   <meta property="og:description" content="{{ page.description }}">
+  <meta property="og:type" content="website">
 {% else %}
   <meta property="og:url" content="{{ site.url }}">
   <meta property="og:title" content="{{ site.title }}">
   <meta property="og:description" content="{{ site.description }}">
 {% endif %}
-<meta property="og:image" content="{{ site.url }}{{ site.social_image_path }}">
+<meta property="og:image" content="{{ site.url | replace: 'https://', 'http://' }}{{ site.social_image_path }}">
```
Comment (mdo) [inline _includes/social.html]: Does the open graph protocol require a non-secure and secure URL? Can we not default to secure?

**Review by mdo — CHANGES_REQUESTED** [review]

File: _includes/footer.html [hunk 4]
```diff
@@ -6,7 +6,7 @@
...
-    <p>Currently v{{ site.current_version }}. Code licensed <a rel="license" href="https://github.com/twbs/bootstrap/blob/master/LICENSE" target="_blank">MIT</a>, docs <a rel="license" href="https://creativecommons.org/licenses/by/3.0/" target="_blank">CC BY 3.0</a>.</p>
+    <p>Designed and built with all the love in the world by <a href="https://twitter.com/mdo" target="_blank" rel="noopener">@mdo</a> and <a href="https://twitter.com/fat" target="_blank" rel="noopener">@fat</a>. Maintained by the <a href="https://github.com/orgs/twbs/people">core team</a> with the help of <a href="https://github.com/twbs/bootstrap/graphs/contributors">our contributors</a>.</p>
+    <p>Currently v{{ site.current_version }}. Code licensed <a rel="license" href="https://github.com/twbs/bootstrap/blob/master/LICENSE" target="_blank" rel="noopener">MIT</a>, docs <a rel="license" href="https://creativecommons.org/licenses/by/3.0/" target="_blank" rel="noopener">CC BY 3.0</a>.</p>
```
Comment (bardiharborow) [inline _includes/footer.html]: Travis is mad because you've got duplicate `rel` attributes on this line. `rel="license noopener"` should work?

**Review by bardiharborow — CHANGES_REQUESTED** [review]

File: _includes/social.html (hunk 3 above)
Comment (bardiharborow) [inline _includes/social.html]: Apparently... http://ogp.me/#structured

**Comment by bardiharborow** [comment]
@XhmikosR sorry to mess with your PR, but this should be good to go now.

**Review by bardiharborow — APPROVED** [review]

File: _includes/footer.html [hunk 5]
```diff
@@ -6,7 +6,7 @@
...
-    <p>Currently v{{ site.current_version }}. Code licensed <a rel="license" href="https://github.com/twbs/bootstrap/blob/master/LICENSE" target="_blank">MIT</a>, docs <a rel="license" href="https://creativecommons.org/licenses/by/3.0/" target="_blank">CC BY 3.0</a>.</p>
+    <p>Designed and built with all the love in the world by <a href="https://twitter.com/mdo" target="_blank" rel="noopener">@mdo</a> and <a href="https://twitter.com/fat" target="_blank" rel="noopener">@fat</a>. Maintained by the <a href="https://github.com/orgs/twbs/people">core team</a> with the help of <a href="https://github.com/twbs/bootstrap/graphs/contributors">our contributors</a>.</p>
+    <p>Currently v{{ site.current_version }}. Code licensed <a rel="license noopener" href="https://github.com/twbs/bootstrap/blob/master/LICENSE" target="_blank">MIT</a>, docs <a rel="license noopener" href="https://creativecommons.org/licenses/by/3.0/" target="_blank">CC BY 3.0</a>.</p>
```
Comment (XhmikosR) [inline _includes/footer.html]: This doesn't make sense @bardiharborow since rel license doesn't have target="_blank". Please revert such changes, thanks.

**Comment by XhmikosR** [comment]
@bardiharborow: Iignore my previous comment. I just reviewed the changes and looks good although personally I wouldn't add `rel="license"` here.

**Comment by bardiharborow** [comment]
@XhmikosR I didn't get your previous comment, but why don't you think the pre-existing `rel="license"` is appropriate? http://microformats.org/wiki/rel-license

**Comment by XhmikosR** [comment]
I thought you added it in this branch. That is why I generally prefer I rebase my PRs to check the changes ;)

File: _includes/social.html (hunk 3 above)
Comment (RinkAttendant6) [inline _includes/social.html]: Nothing on that page says that the `og:image` property can't be an https URL or that it has to be different than `og:image:secure_url`

**Comment by XhmikosR** [comment]
No one said it does. I simply went the explicit way. Both work similarly.

**Review by mdo — APPROVED** [review]

**Pull request merged**